lasso/dyna/ArrayType.py

```diff
@@ -40,12 +40,15 @@ class ArrayType:
     element_solid_effective_plastic_strain = (
         "element_solid_effective_plastic_strain"  #: shape: (n_states, n_solid_layers, n_solids)
     )
-    element_solid_history_variables = "element_solid_history_variables"  #: shape: (n_states, n_solids, n_solid_layers, n_solids_history_vars)
+    #: shape: (n_states, n_solids, n_solid_layers, n_solids_history_vars)
+    element_solid_history_variables = "element_solid_history_variables"

```

Comment:
I haven't gotten to the docs yet but if you put it above then sphinx won't display the helptext. There might be other ways around this though. I would prefer anyway to move to mkdocs.

Reply:
I will check this out and change accordingly